packages/manager/src/features/components/PlansPanel/utils.ts

```diff
@@ -337,11 +345,24 @@ export const extractPlansInformation = ({
 }: ExtractPlansInformationProps) => {
   const plansForThisLinodeTypeClass: PlanWithAvailability[] = plans.map(
     (plan) => {
+      const isMtcTt2025Plan = isMtcTT2025Plan(plan);
```

Comment:
Having the same name for the variable and function might be a little confusing - what do you think about something like this?
```suggestion
      const isPremiumMTCPlan = isMTCTTPlan(plan);
```

Reply:
Or, in case they plan to include it in another category beyond just Premium, we could go with something more flexible like `isCustomMTCPlan` or something similar

Reply:
Or, in case they plan to include it in another category beyond just `Premium`, we could go with something more flexible like `isCustomMTCPlan` or something similar.

Updated to `isCustomMTCPlan` for now to keep it flexible.